pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -44,3 +103,24 @@ var (
 		},
 	}
 )
+
+func RewriteNodeIPs(nodeList *corev1.NodeList) *metadataOptions {
```

Comment:
Using `[]*corev1.Node` allows accurate mocking and makes it possible for consumers to wire in listers.  NodeList contains things like RV of the resource which isn't present in most controller use-cases.